js/views/modals/Test.js

```diff
@@ -0,0 +1,34 @@
+import BaseModal from './BaseModal';
+import loadTemplate from '../../utils/loadTemplate';
+import TestModal2 from './Test2';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+  }
+
+  className() {
```

Comment:
A couple of notable differences in extending from the baseModal (mainly due to some ES6 restrictions). When you declare either a className or events for your view which is extending from the BaseVw, make sure to declare them as functions (ES6 does not support class properties) and make sure to include the value from the base class. For example, notice how in `className` I append `super.className()` to my custom class -and- in events I merge `super.events()` with my own events object.